pkg/jobrunaggregator/jobruntestcaseanalyzer/cmd.go

```diff
@@ -0,0 +1,320 @@
+	}
+	for _, jobPair := range jobPairs {
+		jStrs := strings.Split(jobPair, "=")
+		if len(jStrs) != 2 {
+			return fmt.Errorf("GCS prefix should consist of job name and GCS prefix separated by '='")
+		}
+		*s.values = append(*s.values, jobGCSPrefix{jobName: jStrs[0], gcsPrefix: jStrs[1]})
+	}
+	return nil
+}
+
+func (s *jobGCSPrefixSlice) Type() string {
+	return "jobGCSPrefixSlice"
+}
+
+type stringSlice struct {
+	values []string
+}
+
+func (s *stringSlice) String() string {
+	return strings.Join(s.values, ",")
+}
+
+func (s *stringSlice) Set(value string) error {
+	s.values = append(s.values, value)
+	return nil
+}
+
+func (s *stringSlice) Type() string {
+	return "stringSlice"
+}
+
+type JobRunsTestCaseAnalyzerFlags struct {
+	DataCoordinates *jobrunaggregatorlib.BigQueryDataCoordinates
+	Authentication  *jobrunaggregatorlib.GoogleAuthenticationFlags
+
+	TestGroups                  stringSlice
+	WorkingDir                  string
+	PayloadTag                  string
+	Timeout                     time.Duration
+	EstimatedJobStartTimeString string
+	Platform                    string
+	Infrastructure              string
+	Network                     string
+	MinimumSuccessfulInstall    int
+	PayloadInvocationID         string
+	JobGCSPrefixes              []jobGCSPrefix
+}
+
+func NewJobRunsTestCaseAnalyzerFlags() *JobRunsTestCaseAnalyzerFlags {
+	return &JobRunsTestCaseAnalyzerFlags{
+		DataCoordinates: jobrunaggregatorlib.NewBigQueryDataCoordinates(),
+		Authentication:  jobrunaggregatorlib.NewGoogleAuthenticationFlags(),
+
+		WorkingDir:                  "test-case-analyzer-working-dir",
+		EstimatedJobStartTimeString: time.Now().Format(kubeTimeSerializationLayout),
+		Timeout:                     3*time.Hour + 30*time.Minute,
+	}
+}
+
+const kubeTimeSerializationLayout = time.RFC3339
+
+func (f *JobRunsTestCaseAnalyzerFlags) BindFlags(fs *pflag.FlagSet) {
+	f.DataCoordinates.BindFlags(fs)
+	f.Authentication.BindFlags(fs)
+
+	fs.Var(&f.TestGroups, "test-group", "One or more test groups to analyze, like install")
+	fs.StringVar(&f.PayloadTag, "payload-tag", f.PayloadTag, "The payload tag to analyze test case status, like 4.9.0-0.ci-2021-07-19-185802")
+	fs.StringVar(&f.EstimatedJobStartTimeString, "job-start-time", f.EstimatedJobStartTimeString, fmt.Sprintf("Start time in RFC822Z: %s", kubeTimeSerializationLayout))
+	fs.StringVar(&f.Platform, "platform", f.Platform, "The platform used to narrow down a subset of the jobs to analyze, ex: aws|gcp|azure|vsphere")
+	fs.StringVar(&f.Infrastructure, "infrastructure", f.Infrastructure, "The infrastructure used to narrow down a subset of the jobs to analyze, ex: upi|ipi")
+	fs.StringVar(&f.Network, "network", f.Network, "The network used to narrow down a subset of the jobs to analyze, ex: sdn|ovn")
+	fs.IntVar(&f.MinimumSuccessfulInstall, "minimum-successful-install", 1, "minimum number of successful installations among jobs meeting criteria")
```

Comment:
this is `minimum-successful-count`, right?  versus a possible future `minimum-successful-percent` or `minimum-successful-likelihood`